.pre-commit-config.yaml

```diff
@@ -13,6 +18,7 @@ repos:
         name: Run ansible-test sanity
         entry: ansible-test sanity
         require_serial: true
+        stages: [manual]
```

Comment:
If anyone wonders, these tasks should not be here, too slow and already broken on both pre-commit ci and locally. We will ensure we run them using tox.